apps/next/features/banner/renderBanner.ts

```diff
@@ -0,0 +1,56 @@
+import { TwitchClientAuthService } from "@app/services/TwitchClientAuthService";
+import { remotionAxios, twitchAxios } from "@app/util/axios";
+import { getBannerEntry } from "@app/util/database/postgresHelpers";
+import type { Prisma } from "@prisma/client";
+import type { AxiosResponse } from "axios";
+
+export type TemplateRequestBody = {
+    foregroundId: string;
+    backgroundId: string;
+    foregroundProps: Record<string, unknown>;
+    backgroundProps: Record<string, unknown>;
+};
+
+/**
+ * @param userId
+ * @param twitchUserId
+ * @returns base64 of rendered banner
+ */
+export const renderBanner = async (userId: string, twitchUserId: string): Promise<string> => {
```

Comment:
This should probably also either return a specific string if they are not live or return a different type based on my comment down below.